backend/scripts/create_superuser.py

```
@@ -0,0 +1,34 @@
+import django
+
+def run():
+    # Setup Django
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'core.settings')
+    django.setup()
+
+    from django.contrib.auth import get_user_model
+    from payments.models import Profile
+    User = get_user_model()
+    username = 'admin'
+    password = 'password123'
```

Ruff (0.14.8)

Possible hardcoded password assigned to: "password"

(S105)